l10n_it_amount_to_text/models/res_currency.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 Ecosoft Co., Ltd (http://ecosoft.co.th)
+# Copyright 2022 Sergio Zanchetta (Associazione PNLUG - Gruppo Odoo)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+
+from num2words import num2words
+
+from odoo import models, tools
+
+CURRENCY_NAME = {"Dollars": "dollari", "Euros": "euro", "Cents": "centesimi"}
```

Comment:
"Euros" in quale caso verrebbe usato?
Mi pare che la combinazione IT/EUR sia già gestita esplicitamente nell'implementazione qui sotto.

Inoltre, non si potrebbero riciclare i valori già definiti in
```
>>> from num2words.lang_IT import Num2Word_IT
>>> Num2Word_IT.CURRENCY_FORMS
{'EUR': (('euro', 'euro'), ('centesimo', 'centesimi')), 'USD': (('dollaro', 'dollari'), ('centesimo', 'centesimi')), 'GBP': (('sterlina', 'sterline'), ('penny', 'penny')), 'CNY': (('yuan', 'yuan'), ('fen', 'fen'))}
```
?

Reply:
> "Euros" in quale caso verrebbe usato? Mi pare che la combinazione IT/EUR sia già gestita esplicitamente nell'implementazione qui sotto.
> 
> Inoltre, non si potrebbero riciclare i valori già definiti in
> 
> ```
> >>> from num2words.lang_IT import Num2Word_IT
> >>> Num2Word_IT.CURRENCY_FORMS
> {'EUR': (('euro', 'euro'), ('centesimo', 'centesimi')), 'USD': (('dollaro', 'dollari'), ('centesimo', 'centesimi')), 'GBP': (('sterlina', 'sterline'), ('penny', 'penny')), 'CNY': (('yuan', 'yuan'), ('fen', 'fen'))}
> ``` 
> ?

Questa riga aveva senso nella prima implementazione della PR (prima del merge di https://github.com/savoirfairelinux/num2words/pull/434) perché andava a correggere i valori presenti in https://github.com/odoo/odoo/blob/14.0/odoo/addons/base/data/res_currency_data.xml.
Con la nuova implementazione (vedi sopra) non serve più perchè prende i valori da num2words.